VianneyR4/Free_mentors
# Bg arrange elements PT-id: 168260732

#### What does this PR do?
- arrange some elements (center ...)
- modify some colors
- arrange the footer to the bottom
- add links to access the mentor pages and admin pages

#### Description of Task to be completed
- A user should just enter the URL to access the website

#### How should this be manually tested?
- clone the repo and switch to the branch bg-arrange-elements-168260732
- start your server
- use the url `http://localhost:8080/Free_mentors/UI/html/index.html ...` to access to the website
- you should browse all the website

#### What are the relevant pivotal tracker stories?
[#168260732](https://www.pivotaltracker.com/n/projects/2393639)
